docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,359 @@
+
+## Dashboards and visualizations
+
+### Enhanced data source selection experience: simplifying querying for users
+
+<!-- Ivan Ortega & Natalia Bernarte -->
+
+_Available in preview in all editions of Grafana._
+
+Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data. Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions.
+
+With this flow, selecting a data source has been greatly simplified, providing a clear overview of available data sources and allowing users to quickly connect to a new one when needed. Additionally, the flow now enables quick uploading of CSV files and access to built-in data sources.
+
+{{< video-embed src="/media/docs/grafana/screen-recording-ds-picker-whats-new-10.mp4" max-width="750px" caption="Datasource picker flow" >}}
+
+To try it out, you'll need to enable the `advancedDataSourcePicker` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+### Public dashboards
+
+<!-- Juani Cabanas & Ezequiel Victorero & Natalia Bernarte -->
+
+_Available in preview in all editions of Grafana._
+Public dashboards allow you to share your Grafana dashboard with anyone without requiring them to log in to Grafana. This is useful when you want to make your dashboard available to the world.
+
+With this update, we've made the following improvements:
+
+- The time picker and annotations can be toggled on or off from the configuration modal
+- You can see a list of all your public dashboards at **Dashboards > Public dashboards**
+- Improved UI: new modal design, paused and not found pages
+- Added support for collapsed rows, hidden queries, and zoom into panels
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+To learn more, check out our public dashboards [documentation]({{< relref "../../dashboards/dashboard-public" >}}).
+
+### Public dashboards insights
+
+<!-- Juani Cabanas & Ezequiel Victorero & Natalia Bernarte -->
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+The public dashboards insights provide valuable information about your public dashboard usage. You can easily access and view important metrics such as the daily query count, the number of views in the last 30 days, and the number of errors in the last 30 days.
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+### Email sharing for public dashboards
```

Comment:
@thanos-karachalios  We may want to rephrase this to 
>Email Sharing for Dashboards

The title here could be confused that the dashboard is public, but its also shared via email. Removing public from here could clear this up. Either that or you could mention below that the dashboard is only visible by the people you share it with.

Reply:
cc @evictorero